src/platforms/flutter/usage/advanced-usage.mdx

```diff
@@ -28,6 +28,8 @@ MaterialApp(
 )
 ```
 
+To track automatic Breadcrumbs for `HTTP` requests, Check out the [SentryHttpClient](/platforms/dart/usage/advanced-usage/#automatic-breadcrumbs) Wrapper for the [http](https://pub.dev/packages/http) library.
```

Comment:
```suggestion
To track automatic breadcrumbs for `HTTP` requests, check out the [SentryHttpClient](/platforms/dart/usage/advanced-usage/#automatic-breadcrumbs) wrapper for the [http](https://pub.dev/packages/http) library.
```

Reply:
I'd rather keep it with a capital letter, our docs mention always `B`readcrumbs
https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types